### DISCUSSION TITLE
How to create a Single Author page?

### DISCUSSION THREAD

**Discussion by umangSharmacs** [pacollins/calligraphy#11] [Q&A] [answered]
I am new to Hugo and was wondering how to create just a single 'About the Author' page instead of the current multiple Authors approach that is enabled.

**Comment by pacollins** [comment, marked as answer]
There are a couple of options.

1. [Remove all authors except yourself](https://github.com/pacollins/calligraphy/tree/main/exampleSite/content/authors) and then add `authors/your-name` to the [`config.toml` menu settings](https://github.com/pacollins/calligraphy/blob/2207fab30d716120e47740410eb196989f87e9d2/exampleSite/config.toml#L23-L51). This method can be seen on [my personal site](https://pacollins.com/). 
2. Just use [About Page](https://github.com/pacollins/calligraphy/blob/main/exampleSite/content/about/index.md)

**Reply by umangSharmacs** [reply]
Thank you! This should work quite well.
